gammapy/scripts/tests/test_cta_sensitivity.py

```diff
@@ -0,0 +1,24 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import numpy as np
+from astropy.tests.helper import assert_quantity_allclose
+from numpy.testing import assert_allclose
+from astropy.units import Quantity
+from gammapy.utils.testing import requires_data, requires_dependency
+from .. import cta_sensitivity
+
+
+@requires_dependency('scipy')
+@requires_data('gammapy-extra')
+
+def test_cta_sensitivity():
+    """Test that CTA sensitivity can be well evaluated."""
+
+    irffile = '$GAMMAPY_EXTRA/datasets/cta/CTA-Performance-North-20170327/CTA-Performance-North-20deg-average-30m_20170327.fits'
```

Comment:
This file doesn't exist in `gammapy-extra`. Can you pick an existing file or add this file?